tests/playwright/package.json

```diff
@@ -23,7 +23,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "1.27.1",
```

Comment:
Did the 1.27.1 was breaking anything? Using `carrot` is generally not a good practise since can auto update

Reply:
afedersiniz, I think that was not intentionally! :)